src/components/dashboard/Dashboard.js

```diff
@@ -825,7 +826,17 @@ const Dashboard = props => {
       const scrollPosition = nativeEvent.contentOffset.y
       const { minScrollRequiredISH, scrollPositionGap, isFeedSizeEnough } = scrollData
       const scrollPositionISH = scrollPosition + scrollPositionGap
-      setHeaderLarge(!isFeedSizeEnough || scrollPositionISH < minScrollRequiredISH)
+
+      // because enlarging the feed changes the scroll position (minus), we cannot use this directly for headerLarge
+      // as during slow-scrolling the statement could (depending on platform) jump in-out of being true
+      // why we rely here on a more static ref value
+      if (scrollPositionISH > minScrollRequiredISH) {
+        staticScrollbarOffset.current = 1
+      } else if (scrollPosition < 50) {
+        staticScrollbarOffset.current = 0
+      }
+
+      setHeaderLarge(!isFeedSizeEnough || staticScrollbarOffset.current < 1)
```

Comment:
there's no need for the ref here

Reply:
wdym? what is needed here then?

Reply:
above fix makes for a way smoother transition imo

Reply:
@L03TJ3 it would so, if your calculations were based on the current/previous value stored in the ref. But they aren't. You take the current scrollPosition received from event, and memoized scrollData (which actually is memo acting and changing like state var - maybe you just failed with `scrollData`  or `handleScrollEnd`  deps) ? 

Please

1. revert ref solution
2. review deps of the use memo and use callback - wrapped handler
3. see how do you get other values used in the handler and which functions do you call from there. maybe there are also missing deps causing wrong behaviour
4. **to quickly check** am I right or not - just _remove all useCallback / useMemo_ and _look what's happening_

Reply:
thats all not relevant.

The issue is on android that the position of the scroll bar changes when the header is minimized, causing the position to fallback to the value where it should not minimize.

Reply:
@L03TJ3 again I don't see the ref brings something specific or fixes or changes behaviour in any way. there is no sense in such fix. Please follow the list i've posted

Reply:
there is no 'wrong value'

it just keeps jumping back and forth between above/below threshold as the scrollbar does not keep its offset position when header is changing size

Reply:
its a minor bug, and the solution works pretty good.
I don't see why I should bother spending another hour on this

Reply:
@L03TJ3 I've reverted your "fix". let's discuss the issue you've met tomorrow at the sync and I will suggest you some real solution could be applied

Reply:
I would have liked to have @vldkhh to qa it and see if it fixes anything

Reply:
I am sure about the behavior causing the issue